internal/x11/win/frame.go

```diff
@@ -342,7 +342,6 @@ func (f *frame) drawDecoration(pidTop xproto.Pixmap, drawTop xproto.Gcontext, pi
 		b.SetMaximized(f.client.maximized)
 		// TODO maybe update icon?
 	}
-	f.canvas.SetScale(scale)
```

Comment:
This line is required

Reply:
It looks to be deprecated (`(345, 2)  SA1019  f.canvas.SetScale is deprecated: Settings are now calculated solely on the user configuration and system setup.`). Reverted and ignored for now.